lib/utilities.js

```diff
@@ -1,5 +1,42 @@
 'use strict';
 
+/**
+ * Constants.
+ */
+var CASE_SENSITIVE_TAGS = [
+    'animateMotion',
+    'animateTransform',
+    'clipPath',
+    'feBlend',
+    'feColorMatrix',
+    'feComponentTransfer',
+    'feComposite',
+    'feConvolveMatrix',
+    'feDiffuseLighting',
+    'feDisplacementMap',
+    'feDropShadow',
+    'feFlood',
+    'feFuncA',
+    'feFuncB',
+    'feFuncG',
+    'feFuncR',
+    'feGaussainBlur',
+    'feImage',
+    'feMerge',
+    'feMergeNode',
+    'feMorphology',
+    'feOffset',
+    'fePointLight',
+    'feSpecularLighting',
+    'feSpotLight',
+    'feTile',
+    'feTurbulence',
+    'foreignObject',
+    'linearGradient',
+    'radialGradient',
+    'textPath',
```

Comment:
Omit trailing comma as it is only supported in ES6-compliant browsers

Reply:
That's actually not true, it's supported in IE >= 9 and is part of ES5. [Source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas)

I can still remove them if you'd like.

Reply:
Ah yes, thanks for the correction. Let's omit them for now until I add a linter to enforce this rule.